docker/ubuntu_20_04_base

```diff
@@ -1,5 +1,8 @@
 FROM ubuntu:20.04
 
+ADD . /src
```

Comment:
Why push this line to top? Now the base image can never be re-used (since the code always changes). I'd put this line on top of ubuntu_20_04_lofar.

Reply:
Agreed, the base image should not/need not contain DP3 related source code